wpengine/faustjs
# [MERL-167] FaustWP Enable Usage Tracking Setting

## Description

This PR Includes a new setting for enabling Telemetry in FaustWP. It should be disabled by default. The user should be prompted once, after install, to turn on Telemetry if they haven’t using an admin notice.

## Related Issue(s):



https://wpengine.atlassian.net/browse/MERL-167

## Testing

1. Load plugin
2. Check that the new setting is present in the Settings page and is unchecked by default.
3. Check that when you update the setting it successfully persists in the database.
4. Check that when installing the plugin there is an admin notice for prompting the user to enable Telemetry.
5. Check that when dismissing the prompt, it should not re-appear when disabling the telemetry setting.
6. Check that when deactivating and re-activating the plugin the prompt should re-appear if the telemetry setting is disabled.

## Screenshots


<img width="1269" alt="Screenshot 2022-07-06 at 13 16 28" src="https://user-images.githubusercontent.com/328805/177548045-6277aba4-3c0b-4a7f-bc47-6a81897918cf.png">

<img width="1779" alt="Screenshot 2022-07-07 at 16 02 19" src="https://user-images.githubusercontent.com/328805/177806509-c885713d-5000-41c9-a219-41aa4f328047.png">


## Documentation Changes



## Dependant PRs
